source/CommonJobs/CommonJobs.Domain.MyMenu/Menu.cs

```diff
@@ -22,7 +22,16 @@ public Menu()
         public StringKeyedCollection<Place> Places { get; set; }
         public DateTime StartDate { get; set; }
         public DateTime EndDate { get; set; }
+        /// <summary>
+        /// The date the last order has been generated
+        /// (If we are generating the 01/01/15 the order for the 02/01/15, this field will contain 01/01/15)
```

Comment:
Minor: typo "if we are generating ~~the 01/01/15 the order for 02/01/15~~  the order for 02/01/15 on 01/01/15"